tests/queries/0_stateless/03130_generateSnowflakeId.sql

```diff
@@ -1,12 +1,15 @@
 -- Test SQL function 'generateSnowflakeID'
 
 SELECT bitAnd(bitShiftRight(toUInt64(generateSnowflakeID()), 63), 1) = 0; -- check first bit is zero
-
-SELECT generateSnowflakeID(1) = generateSnowflakeID(2); -- disabled common subexpression elimination --> lhs != rhs
 SELECT generateSnowflakeID() = generateSnowflakeID(1); -- same as ^^
 SELECT generateSnowflakeID(1) = generateSnowflakeID(1); -- enabled common subexpression elimination
 
-SELECT generateSnowflakeID(1, 2); -- { serverError NUMBER_OF_ARGUMENTS_DOESNT_MATCH }
+SELECT generateSnowflakeID(1) = generateSnowflakeID(2); -- disabled common subexpression elimination --> lhs != rhs
+SELECT generateSnowflakeID(1) != generateSnowflakeID(); -- Check different invocations yield different results
+
+SELECT generateSnowflakeID('expr', 1) = generateSnowflakeID('expr', 1); -- enabled common subexpression elimination
```

Comment:
What is missing is:
- a test that checks that the generated snowflake id actually contains the user-provided machine id (the test could be similar to l. 3)
- a test that provides a value of a disallowed type (for example `String`) as machine-id. We can check that a certain error is returned in this case, please grep for `serverError` in other tests in tests/queries/0_stateless for how to do that
- a test that checks what happens if the user-provided machine id is non-const. Use function `materialize` to generate non-const values. I guess it makes sense to enforce that the machine id is const. For this purpose, please specify `isColumnConst` as third argument in the FunctionArgumentDescriptor for the user id and replace `UInt*` by `const UInt*`.

Reply:
Thank you for your response! Changes are made!